packages/web/src/pages/oauth-login-page/OAuthLoginPage.tsx

```diff
@@ -411,14 +684,23 @@ export const OAuthLoginPage = () => {
       setAndLogGeneralSubmitError(false, messages.miscError)
     } else {
       setIsSubmitting(true)
-      authAndRedirect(account)
+      authorize(account)
```

Comment:
do we want to await here?

Reply:
we don't need to because nothing comes after